### PR TITLE
43readable

### PR DESCRIPTION
**Pull request by trel**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -23,12 +20,26 @@ Update for correct version
...
 
 The resulting `site` directory will contain the generated standalone documentation.
 
 
+----
+
+The `Makefile` default target will build two other targets, 'doxygen' and 'mkdocs'.
+
+```
+$ make [MAKEGITHUBACCOUNT=irods] [MAKEIRODSVERSION=master]
```
Comment (alanking) [inline README.md:43]: Should this be `MAKEIRODSVERSION=main`?

**Comment by korydraughn** [comment]
Immediate findings:
- missing indentation on left menu
- missing menu items for each icommand in *user* section
- missing bold styling for icommands help text

**Review by alanking — COMMENTED** [review]
Basically every item in the sidebar navigation which has links is either missing them entirely or they have moved in a surprising way, so those need to be audited in addition the iCommands -> User page.

**Comment by trel** [comment]
Fixed all the navigation... and a bit more.

**Comment by trel** [comment]
todos:
 - top image in doxygen
 - get 4.2.x release notes to main
 - copy/paste paragraphs from blog posts into release notes here
 - new sections for eight capabilities
   - indexing_poc.md ported

**Comment by trel** [comment]
now runs with README instructions...   `docker build` and `docker run`

----

todos:
 - new sections for eight capabilities
     - indexing_poc.md ported

**Review by alanking — COMMENTED** [review]
I'm not qualified to speak on some of the changes but it seems fine overall!

File: README.md [hunk 2]
```diff
@@ -1,17 +1,14 @@
...
 The iRODS documentation is housed in this repository (https://github.com/irods/irods_docs).
 
 Each release is kept in a separate branch.
 
 Prerequisites
 -------------
 
-- git, g++, flex, bison, cmake, python, virtualenv, mkdocs
-- iCommands (the version the generated docs will reflect)
+- docker
```
[the remark this answers is not in the document]
Comment (trel) [reply README.md:11]: i see - will think on it.        i'm also not clear on how to run this in github actions now...

File: README.md [hunk 3]
```diff
@@ -37,9 +37,11 @@ The resulting `site` directory will contain the generated standalone documentati
 
 ----
 
+TGR - this should probably go away...  not sure whether still useful/relevant...
```
[the remark this answers is not in the document]
Comment (trel) [reply README.md:42]: we do not.   will torch it.

File: docs/plugins/pluggable_rule_engine.md [hunk 4]
```diff
@@ -14,7 +14,6 @@ The iRODS Consortium currently supports the following rule engine plugins:
  - [C++ Logical Quotas Rule Engine Plugin](https://github.com/irods/irods_rule_engine_plugin_logical_quotas)
```
[the remark this answers is not in the document]
Comment (trel) [reply docs/plugins/pluggable_rule_engine.md:14]: ah yes, will remove.

File: docs/release_notes.md [hunk 5]
```diff
@@ -2,11 +2,11 @@
...
-Release Date: 2019-XX-XX
+Release Date: 2022-XX-XX
 
 ### Notable Features
 
-  - iRODS now requires C++17
+  - iRODS now requires C++20
 
   - New libraries, in use in the server and available to developers
       - irods::filesystem
```
[the remark this answers is not in the document]
Comment (trel) [reply docs/release_notes.md:12]: definitely - this effort is largely to get this thing buildable by others.

**Pull request merged**